components/dialpad/sources/new-call-event/new-call-event.ts

```diff
@@ -0,0 +1,27 @@
+import { defineSource } from "@pipedream/types";
+import constants from "../../common/constants";
+import common from "../../common/common";
+
+export default defineSource({
+  ...common,
+  type: "source",
+  name: "New Call Event",
+  key: "dialpad-new-call-event",
+  description: `Emit new call event subscription. 
+  See [Event doc](https://developers.dialpad.com/reference/webhook_call_event_subscriptioncreate)
+  and [webhook doc](https://developers.dialpad.com/reference/webhookscreate)`,
+  version: "0.0.1",
+  methods: {
+    ...common.methods,
+    getPath() {
+      return constants.EVENT_TYPE.NEW_CALL.path;
+    },
+    processEvent(event) {
+      const { body } = event;
+      this.$emit(body, {
+        id: body.id,
+        summary: `New call event - ${body.id}`,
```

Comment:
We should always emit a timestamp. If the event returns a timestamp of when the event happened, you can use that. Otherwise, you can use `Date.now()`. This applies to the rest of the sources as well.
```suggestion
        id: body.id,
        summary: `New call event - ${body.id}`,
        ts: Date.now(),
```